examples/with-pubnub/next.config.js

```diff
@@ -1,6 +1,20 @@
+const { NormalModuleReplacementPlugin } = require("webpack");
+
 /** @type {import('next').NextConfig} */
-const nextConfig = {
+module.exports = {
   reactStrictMode: true,
+  webpack: (config, { isServer }) => {
+    if (!isServer) {
+      config.resolve.fallback.fs = false;
+      config.resolve.fallback.dns = false;
+      config.resolve.fallback.net = false;
+    }
+    config.plugins.push(
+      new NormalModuleReplacementPlugin(
+        /^hexoid$/,
```

Comment:
Is this necessary? I'd prefer to keep the next config as simple as possible so devs don't have to make these changes when they build with this example

Reply:
I have reported it as a bug to our engineering team to fix inside the SDK. I will check the progress on that. Hopefully, (not likely) we will have it removed before the Webinar. It is not a breaking change it just throws an error in the console if it is not in there which I did not like the look of.